src/lib/lldpctl.h

```diff
@@ -686,6 +686,7 @@ typedef enum {
 	lldpctl_k_port_id_subtype, /**< `(IS)` The subtype ID of this port.  */
 	lldpctl_k_port_id,	   /**< `(BS,WO)` The ID of this port. */
 	lldpctl_k_port_descr,	   /**< `(S,WO)` The description of this port. */
+	lldpctl_k_port_lldp_snap_enabled,	   /**< `(I,WO)` The lldp snap encapsulation enable status of this port. */
```

Comment:
Do not put a new value in the middle of existing values. It breaks the ABI (old clients cannot talk to new lldpd). Add it at the end of the section.